app/models/post.rb

```diff
@@ -1,2 +1,5 @@
 class Post < ActiveRecord::Base
+  validates :title, presence: true
```

Comment:
You can bundle all of these validations into one, like:

`validates :title, :body, :author, presence: true`